WcaOnRails/app/models/delegate_report.rb

```diff
@@ -27,6 +27,7 @@ def equipment_default
   validates :schedule_url, url: true
   validates :discussion_url, presence: true, if: :schedule_and_disussion_urls_required?
   validates :discussion_url, url: true
+  validates :wrc_incidents, presence: true, if: :wrc_feedback_requested
```

Comment:
Just curious: did you consider adding any validations on this field? You could imagine requiring it to always be a comma separate list of numbers, for instance. I'm not saying we *should* do any validation (it could be confusing to the user unless the error message is done well), I'm just wondering if you considered anything.

Reply:
I thought about this, but I'm not sure it's the best move. I feel like I've seen many delegate reports where there hasn't been a numerical list; for example, I want to say I've seen in the past where incidents have been sectioned off by the delegate who's reporting, so an incident would be referenced with "A1" or "B2".